src/hacbs/components/ImportForm/utils/submit-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { FormikHelpers } from 'formik';
+import { createComponents } from '../../../../components/ImportForm/utils/submit-utils';
+import { ImportFormValues } from '../../../../components/ImportForm/utils/types';
+import { createApplication } from '../../../../utils/create-utils';
+import { FormValues } from '../types';
+
+export const onComponentsSubmit = async (
+  { components, applicationData, namespace, secret }: FormValues,
+  formikBag: FormikHelpers<ImportFormValues>,
+) => {
+  try {
+    await createComponents(components, applicationData.metadata.name, namespace, secret, true);
```

Comment:
Components aren't created with dry run. But we should first perform a dry run followed by the real run.
See how app-studio does this in `submit-utils#createResources`

Reply:
Updated it.